spec/ruby/core/main/using_spec.rb

```diff
@@ -129,4 +129,12 @@ def bar; 'quux'; end
 
     x.call_bar(cls2.new).should == 'bar'
   end
+
+  ruby_version_is "3.2" do
+    it "does not raise error when wrapped with module" do
+      -> do
+        load File.expand_path('../fixtures/using.rb', __FILE__), true
+      end.should_not raise_error
+    end
+  end
```

Comment:
The issue is reproduced on 2.7, 3.0 and 3.1. So this `ruby_version_is` guard is excessive.

Reply:
Ok removed the guard :+1:

Reply:
@andrykonchin Reading the documentation on contributing, it seems like in this case I should be using `ruby_bug`, so I've done that so CI passes on 3.2, then it can be backported after. Let me know if I should do something different.

Reply:
Yes, you are right. Looks like `ruby_bug` is the best way 👍

Reply:
Probably I need to improve the wording in `spec/ruby/CONTRIBUTING.md` then.
In general `ruby_bug` should only be used if it's likely the fix will be backported.
Here it seems rather unlikely, so `ruby_version_is "3.2" do` is better (i.e., don't expect any Ruby < 3.2 to implement this).